deploy/examples/cluster-test.yaml

```diff
@@ -37,6 +37,10 @@ spec:
   mgr:
     count: 1
     allowMultiplePerNode: true
+  logCollector:
```

Comment:
Was this just for your testing? It doesn't hurt in the test cluster, but doesn't seem necessary either.

Reply:
ready to revert this?

Reply:
will revert once, I test the latest changes

Reply:
```yaml sh-4.4# cat /etc/logrotate.d/ceph 
/var/log/ceph/*-client.rbd-mirror*.log {
    rotate 28
    daily
    maxsize 500M
    compress
    sharedscripts
    postrotate
        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
    endscript
    missingok
    notifempty
    su root ceph
}
```

```yaml
/var/log/ceph/ceph-osd.0.log {
    rotate 7
    daily
    maxsize 500M
    compress
    sharedscripts
    postrotate
        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
    endscript
    missingok
    notifempty
    su root ceph
}
```

Reply:
removed